docs/src/content/docs/guides/split-payments.mdx

```diff
@@ -0,0 +1,331 @@
+---
+title: Split payments
+---
+
+import { CodeBlock, LinkOut } from '@interledger/docs-design-system'
+import { Tabs, TabItem } from '@astrojs/starlight/components'
+import HttpSig from '/src/partials/http-msg-sig-note.mdx'
+import ChunkedSnippet from '/src/components/ChunkedSnippet.astro'
+import Ts from '/src/partials/ts-prerequisites.mdx'
+
+# Build an e-Commerce platform with split-payments between merchants and platform
+
+This tutorial guides you through building an e-Commerce platform using Open Payments where split-payments are initiated by a buyer of a good or service and split between the merchant and you, the platform operator. Airbnb and Uber are popular platforms where payments made by guests and riders are split between the platform on one side and hosts and drivers on the other respectively.
```

Comment:
Let's work on this. A few points to consider:
- The tutorial doesn't guide you through building an e-Commerce platform. 
- It could be interpreted that the buyer is initiating the split, when in fact the buyer likely has no idea that a split will even occur.
- I'm going to post a question in the user-docs channel about the use of Airbnb, Uber, or any other company.

Reply:
I think it would be handy to list out the three parties here in bullet points